app/services/delivery_request_service.rb

```diff
@@ -18,42 +18,39 @@ def self.call(*args)
   end
 
   def call(email:)
-    raise ArgumentError, "email cannot be nil" if email.nil?
-
     subject = "#{subject_prefix}#{email.subject}"
-    reference = generate_reference(email)
+    reference = SecureRandom.uuid
     address = determine_address(email, reference)
 
-    MetricsService.email_send_request(provider_name) do
-      provider.call(
-        address: address,
-        subject: subject,
-        body: email.body,
+    return if address.nil?
```

Comment:
This is interesting - previously, if we got here with a `nil` email, I think we would have ended up with an error in Sentry; Notify would have errd, we would have caught a the `ProviderError` and then attempted to write a DeliveryAttempt with a nil email, which would fail the validation on the model.  I'm not sure that was ever a conscious decision when we wrote the code or an accidental side effect.